### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by jacobwindsor**
- Increase padding of table
- top left algin cell count

### CODE REVIEW

File: src/app/projects/pages/projects-list/projects-list.component.html [hunk 1]
```diff
@@ -104,7 +104,8 @@
         </td>
 
         <td class="vf-table__cell">
-          <p>{{ project.cellCount ? (project.cellCount | number) : "-" }}</p>
+          <span *ngIf="project.cellCount">{{ (project.cellCount | number) }}</span>
```
Comment (ke4) [inline src/app/projects/pages/projects-list/projects-list.component.html:107]:
```suggestion
         <span *ngIf="project.cellCount; else noCellCount">{{ (project.cellCount | number) }}</span>
         <ng-template #noCellCount>-</ng-template>
```
and remove the next line

File: src/app/projects/pages/projects-list/projects-list.component.html [hunk 2]
```diff
@@ -104,7 +104,8 @@
...
+          <p *ngIf="!project.cellCount">-</p>
```
Comment (ke4) [inline src/app/projects/pages/projects-list/projects-list.component.html:108]:
```suggestion
          <span *ngIf="project.cellCount; else noCellCount">{{ (project.cellCount | number) }}</span>
         <ng-template #noCellCount>-</ng-template>
```

**Review by ke4 — APPROVED** [review]
Looks fine with me.
I added a suggestion, but I am not sure if that much improves the original code.

File: src/app/projects/pages/projects-list/projects-list.component.html (hunk 2 above)
Comment (jacobwindsor) [reply src/app/projects/pages/projects-list/projects-list.component.html:108]: It needs to be a paragraph or a span element in each case. I think your suggestion actually adds more lines. Normally, I would go for this if we didn't need different elements

**Review by MightyAx — APPROVED** [review]

**Pull request merged**